### PR TITLE
More profile (channels) QA

### PR DESCRIPTION
**Pull request by broskoski**
Closes most of the items in #530

### CODE REVIEW

File: apps/user/profile/index.coffee [hunk 1]
```diff
@@ -61,7 +67,7 @@ module.exports.init = ->
   user = new User USER
 
   view = new ProfileView
-    el: $('.profile')
+    el: $('.UserChannels__groups')
```
Comment (dzucconi) [inline apps/user/profile/index.coffee]: Pretty please use `js-` scoped *only*. It really makes refactoring very difficult when you're not sure what has js attached to it other wise.

File: apps/user/stylesheets/user_channels.styl [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.UserChannels
+  margin-top 3em
+  position relative
+  padding-bottom 200px
```
Comment (dzucconi) [inline apps/user/stylesheets/user_channels.styl]: :gasp: px

File: components/block_v2/stylesheets/index.styl [hunk 3]
```diff
@@ -105,14 +108,17 @@
...
       padding unit
       overflow hidden
       text-align left
+      p:first-of-type:empty
+        display none
       p
-        font-family font_serif 
-        p()
+        font-family font_serif
+        font-size 1rem
```
Comment (dzucconi) [inline components/block_v2/stylesheets/index.styl]: `font-size-base`

File: components/block_v2/stylesheets/index.styl [hunk 4]
```diff
@@ -105,14 +108,17 @@
...
+        margin 0.75em 0
+        line-height 1.6em
```
Comment (dzucconi) [inline components/block_v2/stylesheets/index.styl]: `line-height-base`

File: components/block_v2/templates/block.jade [hunk 5]
```diff
@@ -15,9 +15,13 @@ a.Block(
...
-        if (block.source && block.source.url) || block.kind.file_url
-          .Block__inner__overlay__button.js-source Source
-        .Block__inner__overlay__button.js-connect Connect →
+        if block.kind.source_url || block.kind.file_url
+          button.Button.Button--overlay.js-source Source
+        button.Button.Button--overlay.js-connect Connect →
+      else
+        .Block__inner__overlay__channel-actions
+          button.Button.Button--divider.js-follow(class="Button--#{block.kind.visibility}") Follow
+          button.Button.Button--divider.js-connect(class="Button--#{block.kind.visibility}") Connect →
```
Comment (dzucconi) [inline components/block_v2/templates/block.jade]: Oo I haven't see what this looks like yet

File: components/block_v2/view.coffee [hunk 6]
```diff
@@ -28,22 +30,41 @@ module.exports = class BlockView extends Backbone.View
...
+      model = new Channel id: @block.id
+    else
+      model = new Block id: @block.id
+    
+    model
+
   openConnect: (e) ->
     e.preventDefault()
-    e.stopPropagation()
+    e.stopImmediatePropagation()
```
Comment (dzucconi) [inline components/block_v2/view.coffee]: 😮

File: components/block_v2/view.coffee [hunk 7]
```diff
@@ -28,22 +30,41 @@ module.exports = class BlockView extends Backbone.View
...
 
-    @$el.addClass 'Block--is_connecting'
+    $target = @$('.Block__inner__connect')
```
Comment (dzucconi) [inline components/block_v2/view.coffee]: jsssssss

File: components/block_v2/view.coffee [hunk 8]
```diff
@@ -28,22 +30,41 @@ module.exports = class BlockView extends Backbone.View
...
 
     # temp: get a real block
-    block = new Block id: @block.id
+    block = @getModel()
 
     block.fetch 
       success: =>
+        @$el.addClass 'Block--is_connecting'
```
Comment (dzucconi) [inline components/block_v2/view.coffee]: Sup with the underscore

Comment (dzucconi) [inline components/block_v2/view.coffee]: Also typically modifiers don't really need the `is` prefix anymore in BEM-world

**Comment by broskoski** [comment]
Gonna marge!

**Pull request merged**